### ISSUE TITLE
Using `path_prefix` leads to 404 errors when fetching asset during local dev

### ISSUE THREAD

**Issue by pomdtr** [axodotdev/oranda#265]
<img width="1592" alt="image" src="https://user-images.githubusercontent.com/17577332/232330682-cffeb99c-e019-4b3c-be73-e54a2b7f7ddc.png">

**Comment by ashleygwilliams** [comment]
yeah this is a vaguely known issue (see also #184 #227) - it's from some fundamental limitations of axum, the library we are using to do the server. 

https://github.com/axodotdev/oranda/blob/main/src/commands/serve.rs#L78
this is how i'm handling it today which is very manual and will be difficult for custom assets. will need to figure out a different solution.

a workaround is that while developing, you can remove the path prefix and it should work perfectly. then add the path prefix for deploy. not ideal, i know, but should help unblock.

**Comment by pomdtr** [comment]
> a workaround is that while developing, you can remove the path prefix and it should work perfectly. then add the path prefix for deploy. not ideal, i know, but should help unblock.

Maybe path_prefix should be ignored when running `oranda dev` ? I don't really care if my app is running on `localhost:7979` or `localhost:7979/sunbeam`

**Comment by Gankra** [comment]
short-term plan: disabled path-prefix in `serve` environments to avoid this issue
